ansible/install_mariadb.yaml

```diff
@@ -41,8 +37,7 @@
 
   - name: Wait for deployment
     shell: |
-      oc wait -n openstack deployment/mariadb \
-          --for condition=available --timeout=60s
+      oc wait -n openstack pod/mariadb --for condition=ready --timeout=60s
```

Comment:
NOTE: the operator currently uses a POD instead of the deployment here. Also, now that we have MariaDBSchema (which waits on the DB to initialize before continuing) we can soon remove this wait check once we update KeystoneOperator and the like to use that instead of manually creating their own DBs...